data/cassSchema.cql

```diff
@@ -0,0 +1,36 @@
+DROP KEYSPACE IF EXISTS menus;
+
+CREATE KEYSPACE menus
+  WITH REPLICATION = { 
+   'class' : 'SimpleStrategy', 
+   'replication_factor' : 1 
+  };
+
+USE menus;
+
+CREATE TABLE restaurants (
```

Comment:
You could make this 'CREATE TABLE IF NOT EXISTS restaurants' to avoid errors when you re-use or alter your schema in the future, not all that important now though since you've already seeded your db.